components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -173,6 +173,15 @@ class AppLinksUseCases(
 
             return RedirectData(appIntent, fallbackIntent, marketplaceIntent, resolveInfo)
         }
+
+        private fun isPackageInstalled(packageName: String): Boolean {
+            return try {
+                context.packageManager.getPackageInfo(packageName, PackageManager.GET_ACTIVITIES)
+                true
+            } catch (e: PackageManager.NameNotFoundException) {
+                false
+            }
+        }
```

Comment:
Maybe can share this logic as an extension function as we are doing the same on other components [see](https://github.com/mozilla-mobile/android-components/blob/4202c70e488702cc9110b96997f5a5fc3860f7f7/components/support/utils/src/main/java/mozilla/components/support/utils/Browsers.kt#L232)

Reply:
Sure, I'll check if there's a good way to share this.  Thanks,

Reply:
👍

Reply:
Added the extension function.  Unfortunately `support-ktx` depends on `support-utils` causing a circular dependency so I couldn't replace the `Browsers.kt` call.